app/routes/helpers.py

```diff
@@ -0,0 +1,15 @@
+from app.models.planet import Planet
+from flask import abort, make_response, jsonify
+
+# helper function to validate
+def validate_planet(planet_id):
+    try:
+        planet_id = int(planet_id)
+    except:
+        return abort(make_response(jsonify(f"Planet {planet_id} is invalid"), 400))
+
+    planet = Planet.query.get(planet_id)
+
+    if not planet:
+        return abort(make_response(jsonify(f"Planet {planet_id} does not exist"), 404))
+    return planet
```

Comment:
Helpers look great! 👍 We may want to consider making the validation code as generic as possible to utilize any class and id or renaming this file to `helper_planet_routes` or `planet_routes_helper`. Either would be helpful if we expand this project to validate other objects like moons, suns, etc.

Reply:
A more generic validation helper method would look like:

```python
from flask import make_response, abort

def validate_object(cls, id):
    try:
        id = int(id)
    except:
        return abort(make_response({"message": f"{cls} {id} is invalid"}, 400))

    obj = cls.query.get(id)
    
    if not obj:
        abort(make_response({"message": f"{cls} {id} not found"},404))
    
    return obj
```